src/ui/widgets/FormulaWidgetUI.js

```diff
@@ -60,14 +60,14 @@ function FormulaWidgetUI(props) {
     <Box className={classes.root}>
       {typeof formattedValue === 'object' && formattedValue !== null ? (
         <span>
-          <span className={`${classes.unit} ${classes.before}`}>
+          <span aria-label="preffix" className={`${classes.unit} ${classes.before}`}>
```

Comment:
"prefix" ? :roll_eyes:

Reply:
Yeah, typo. Could you see if this has been fixed in your later PRs please?

Reply:
It is not fixed!